styleguide.config.js

```diff
@@ -39,16 +39,10 @@ module.exports = {
             components: [
                 "packages/wonder-blocks-form/components/checkbox.js",
                 "packages/wonder-blocks-form/components/radio.js",
-            ],
-            sections: [
```

Comment:
Oh, it's probably ok to keep these internal components, right? (Along with the internal choice component, too.)